### DISCUSSION TITLE
Enhancement: Add a changelog screen to the main menu.

### DISCUSSION THREAD

**Discussion by objectionist** [FunkinCrew/Funkin#1670] [Ideas]
## What is your suggestion, and why should it be implemented?

As the title states, I suggest adding a changelog screen to the main menu with all the new features and bug fixes as a way to document changes in new updates. This keeps the players up to date, as well as on track. 

While there are patch notes on the desktop version of Friday Night Funkin', we don't always look there. In fact, I'm pretty sure only the even _mildly_ curious people would scrounge through the files to see what's going on behind the scenes.

Something like this would be nice.

![image psd](https://user-images.githubusercontent.com/86072496/122446090-e37af780-cf67-11eb-8f7e-7ad9d5c2a2a7.png)

The actual screen _itself_ is up to you guys.

**Comment by libauror-a** [comment]
This was actualy feature while testing in FreeplayState line 140, this can actually be a good idea.
![image](https://user-images.githubusercontent.com/75984453/122552300-6cf8fb00-d036-11eb-957e-939d07fd777c.png)

**Comment by objectionist** [comment]
> 
> 
> This was actualy feature while testing in FreeplayState line 140, this can actually be a good idea.
> ![image](https://user-images.githubusercontent.com/75984453/122552300-6cf8fb00-d036-11eb-957e-939d07fd777c.png)

Awesome! This just gives more a reason to add it. If it was in consideration, and scrapped because of timeframes, then they certainly have the time to add it now!

**Comment by ninjamuffin99** [comment]
yea like dis commented out code says, i have been thinkin about it. this code was probably written in november or december!! but also i think its cooler for people to discover changes themselves heheh
